python/eos/analysis.py

```diff
@@ -375,21 +375,46 @@ def sample(self, N=1000, stride=5, pre_N=150, preruns=3, cov_scale=0.1, observab
 
     def sample_pmc(self, log_proposal, step_N=1000, steps=10, final_N=5000, rng=np.random.mtrand, return_final_only=True, final_perplexity_threshold=1.0):
         """
-        Return samples of the parameters and log(weights)
+        Return samples of the parameters and log(weights), and a mixture density adapted to the posterior.
```

Comment:
The following lines 380/381 are a better general description. The return value is discussed later. Remove this line?

Reply:
Agreed!